src/dataset-builder/ConceptCart.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { Fragment } from 'react';
+import { ACTION_BAR_HEIGHT, ActionBar } from 'src/components/ActionBar';
+import { SnapshotBuilderConcept as Concept } from 'src/libs/ajax/DataRepo';
+
+interface ConceptCartProps {
+  cart: Concept[];
+  onClick: () => void;
+  actionText: string;
+}
+
+export const ConceptCart = (props: ConceptCartProps) => {
+  const { cart, onClick, actionText } = props;
+
+  return (
+    cart.length > 0 && (
```

Comment:
Was there a reason to change it from `!==` to `>`? I prefer not using `<` or `>` if possible since I think `==` and `!=` are easier to reason about when reading code. Since we know that `.length` cannot be negative I don't see a benefit to use `>` here.